### ISSUE TITLE
Add script for basic testing

### ISSUE THREAD

**Issue by Ricocotam** [Cadene/bootstrap.pytorch#18]
## Feature
Add a script for some basic testing such as finding batch size, testing just the dataset or just the network.

## Motivation
When we are developing a new model we want to test parts of our pipeline before pursuing coding. Having a tool to just test some parts (e.g. : dataset, network creation/forward) and some setting tests before starting training (e.g. : batch size) would ease the use of the framework.

## Example of use
I imagine something like :
- `python -m bootstrap.test --batch_size 50 80 100 --options /path/to/options`
- `python -m bootstrap.test --network forward --options /path/to/options`
- `python -m bootstrap.test --dataset --options /path/to/options`

To test networks we would need datasets to generate empty items (so we don't need to load an entire dataset) so we can feed the network with batches really fast.

**Comment by Cadene** [comment]
@Ricocotam 
After discussing with @MicaelCarvalho, we think that it is a great idea.
For instance, it would be useful to test that our model is able to overfit a small of the dataset, or to [check the gradient](https://towardsdatascience.com/how-to-debug-a-neural-network-with-gradient-checking-41deec0357a9).
However, it's not a priority for now to us.
Feel free to suggest any ideas.

Best
